learning_journal_basic/learning_journal_basic/requirements.txt

```diff
@@ -0,0 +1,83 @@
+ackermann==0.1
```

Comment:
This probably isn't actually used by your learning journal... check your requirements to determine what is necessary...